client/config-overrides.js

```diff
@@ -0,0 +1,17 @@
+const { override, fixBabelImports, addLessLoader, addWebpackPlugin } = require('customize-cra');
+const AntdDayjsWebpackPlugin = require('antd-dayjs-webpack-plugin');
+
+module.exports = override(
+    fixBabelImports('import', {
+        libraryName: 'antd',
+        libraryDirectory: 'es',
+        style: true,
+    }),
+    // Themes
+    addLessLoader({
+        javascriptEnabled: true,
+        modifyVars: { '@primary-color': '#1DA57A' },
+    }),
+    // Reduces timestamp bundle size
+    addWebpackPlugin(new AntdDayjsWebpackPlugin()),
+);
```

Comment:
What does this section do?

Reply:
@ankitjena We will use timestamps for so many things, such as event deadlines relative to timezones. For that, using moment.js is a good option. However `moment.js` is about 600 Kb. So, `day.js` would be about 16 Kb, reducing bundle size significantly